### PR TITLE
MOTECH-2766 Fix column widths in motech-list

### PR DESCRIPTION
**Pull request by jhopen**

### CODE REVIEW

File: src/common/motech-list/list.directive.js [hunk 1]
```diff
@@ -102,27 +102,47 @@
...
 
-        function setColumnWidth(col){
+        function setColumnWidth(col) {
             var widest = 0;
             if(jQuery('.motech-list-rows [motech-list-column-id="'+col.id+'"]', element).parent().hasClass('list-item-main')){
                 return false;
             }
-            jQuery('.motech-list-rows [motech-list-column-id="'+col.id+'"]', element).each(function(){
+
+            jQuery('.motech-list-rows [motech-list-column-id="'+col.id+'"]',element).each(function(){
```
Comment (jredlarski) [inline src/common/motech-list/list.directive.js]: @jhopen Add space before `element`

File: src/common/motech-list/list.directive.js [hunk 2]
```diff
@@ -102,27 +102,47 @@
...
+                if(!cell.is('nav')){
+                    cell = cell.parent();
+                }
                 if(cell.outerWidth() > widest){
-                    widest = cell.outerWidth() + 1;
+                    widest = cell.outerWidth();
+                }
+            });
+
+            var header = jQuery('.motech-list-column-headers [motech-list-column-id="'+col.id+'"]',element).outerWidth();
```
Comment (jredlarski) [inline src/common/motech-list/list.directive.js]: Add space before `element`

File: src/common/motech-list/list.directive.js [hunk 3]
```diff
@@ -102,27 +102,47 @@
...
+            if(header > widest) {
+                widest = header;
+            }
+
+            jQuery('.motech-list-rows [motech-list-column-id="'+col.id+'"]',element).each(function(){
```
Comment (jredlarski) [inline src/common/motech-list/list.directive.js]: Add space before `element`

**Comment by jhopen** [comment]
@jredlarski spaces added

**Comment by jredlarski** [comment]
@jhopen Looks good! I've got only a problem with the last column being as wide as possible, like in Queues:
queue

This should look more simillary to the bundles list in Manage Bundles:
bundles
So the first column should be the column which fills the rest of the available space. This can be done by changing the first column ("Name" column) html element from `<motech-list-column>` to simple `<div>` in [queues-list.html](https://github.com/motech/motech-ui/blob/master/src/admin/queues/queues-list.html#L3) and [topics.html](https://github.com/motech/motech-ui/blob/master/src/admin/topics/topics.html#L3). This is slightly out of scope of your ticket but should be a really quick fix, so please add it to this PR.

**Comment by jhopen** [comment]
@jredlarski  fix: done

**Pull request merged**